home.md

```diff
@@ -1,24 +1,57 @@
-Changing the provider will update all of the information shown within the inputs
-application. You will immediately see the map update showing the selected
-provider, as well as that trusts peers, as defined by the
-[Trust Peer Finder Tool][1]. The list of these peers is displayed below the
-map (collapsed by default). Clicking on any of the points on the map will reveal
-the name of the provider.
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
 
-[1]: https://app.powerbi.com/view?r=eyJrIjoiMjdiOWQ4YTktNmNiNC00MmIwLThjNzktNWVmMmJmMzllNmViIiwidCI6IjUwZjYwNzFmLWJiZmUtNDAxYS04ODAzLTY3Mzc0OGU2MjllMiIsImMiOjh9
+::: callout-note
```

Comment:
This is a Markdown file, so callouts won't work here. The simplest thing to do is just remove the fences (so this will just appear as a standalone paragraph with no special markup).
```suggestion
```